Library/Styles/ButtonStyles.swift

```diff
@@ -101,6 +101,20 @@ public let navyButtonStyle =
     <> UIButton.lens.layer.borderColor .~ UIColor.ksr_navy_900.cgColor
     <> UIButton.lens.layer.borderWidth .~ 1.0
 
+public let saveButtonStyle =
```

Comment:
this and `shareButtonStyle` are some great shared styles!

whatdya think about using these in the project nav bar view too so that we have a single place we are stylin?